functions/go/set-default-labels/README.md

```diff
@@ -0,0 +1,54 @@
+# set-default-labels
+
+## Overview
+
+<!--mdtogo:Short-->
+
+The `set-default-labels` function applies KPT label convention to your KPT package.
+
+
+### `app.kubernetes.io/name` 
+
+Running the function can add this label using the value from the root `Kptfile` name.
+
+This convention believes that all KRM resources under the same kpt package should be served for a specific application.
+An application (or KPT package) can be composed by other applications (nested sub KPT packages). 
+So the function can accept multiple Kptfile and use the root Kptfile to set the app name.
+
+<!--mdtogo-->
+
+You can learn more about recommended labels [here][recommended labels].
+
+## Usage
+
+This function should be run in a KPT package. It does not require function config.
+
+### Run the function once
+```bash
+kpt fn eval --image set-default-labels
+```
+
+### Run the function as a Kpt pipeline
+```bash
+# add the function to Kptfile
+kpt fn eval -t mutator -s -i set-default-labels:unstable
+
+# check Kptfile
+cat Kptfile
```

Comment:
Each command should be in its own code block. Then the comment can be moved out of the code block as Markdown text.